packages/dashboard/src/components/DashboardGridLayout/DashboardGridLayoutWithAddCardPanel/DashboardGridLayoutWithAddCardPanel.tsx

```diff
@@ -42,7 +42,8 @@ export class DashboardGridLayoutWithAddCardPanel extends BaseComponent<
       selectedCardId: '',
       selectedCardInitialX: 0,
       selectedCardSize: CardSize.small,
-      selectedCardTitle: ''
+      selectedCardTitle: '',
+      cardNodes: [<div key="section0" />]
```

Comment:
> [<div key="section0" />] [](start = 16, length = 25)

can you actually create a proper card as an example instead of putting this. So, that people use that when creating

Reply:
Updated the example.